File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/10.java
public class AFNZeroSeguidoUm {
    public static boolean zeroSeguidoUm(String s) {
        return s.contains("01");
    }

    public static void main(String[] args) {
        System.out.println(zeroSeguidoUm("101"));  // True
        System.out.println(zeroSeguidoUm("111"));  // False
    }
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/10.go
package main
import "fmt"
import "strings"

func zeroSeguidoUm(s string) bool {
    return strings.Contains(s, "01")
}

func main() {
    fmt.Println(zeroSeguidoUm("101"))  // True
    fmt.Println(zeroSeguidoUm("111"))  // False
}
